src/main/java/frc/robot/subsystems/ShooterSubsystem

```diff
@@ -0,0 +1,119 @@
+// INTAKE one talon for feeding one neo for  pivot
+// SHOOTER one neo for shooting, one neo for pivot, one talon for conveyer belt
+
+//create enums, expecting, holding, firing, and no note
+
+package frc.robot.subsystems;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+//enum, change decleration location later
+enum robotState {
+    LOADING_NOTE, //taken through intake
+    HOLDING_NOTE, //in the conveyer belt
+    NO_NOTE, //no note in robot
+    FIRING
+}
+
+public class ShooterSubsystem extends SubsystemBase {
+
+    //change later
+    int IDNUMBER = 10; //so I remember to change them later
+    double speed = 0.1; 
```

Comment:
if this is a constant, capitalize it, otherwise move it to a different part of the variable declarations